rococo-parachains/src/service.rs

```diff
@@ -126,6 +126,7 @@ where
 
 	let params = new_partial(&parachain_config)?;
 	let telemetry_span = params.other;
+	let _telemetry_span_entered = telemetry_span.as_ref().map(|x| x.enter());
```

Comment:
Why do we need to do this here manually?

Reply:
I need to investigate but I think there is something in a function somewhere below that reports something to the telemetry.

I know it's vague and it's probably not needed for the telemetry to work. I just wanted to make sure we don't miss a telemetry message at all.

Reply:
Please investigate this. The problem here is that this isn't any code that is reusable, so people would need to copy this.

Reply:
Fixed by https://github.com/paritytech/substrate/pull/7951